### PR TITLE
index/jump-diffusion

### PR DESCRIPTION
**Pull request by arash-binary**

### CODE REVIEW

File: share/underlyings.yml [hunk 1]
```diff
@@ -2795,3 +2795,78 @@ RB200:
   symbol: 1HZ75V
   market_type: non_financial
   generation_interval: 1s
+JD10:
+  asset: JD10
+  display_name: Jump-Diffusion 10
```
Comment (junbon-binary) [inline share/underlyings.yml:2800]: Do we have a `-` in our index naming convention?

File: share/underlyings.yml [hunk 2]
```diff
@@ -2795,3 +2795,78 @@ RB200:
...
+  exchange_name: RANDOM
+  instrument_type: stockindex
+  market: synthetic_index
+  market_convention:
+    delta_premium_adjusted: 0
+    delta_style: spot_delta
+  pip_size: 0.01
+  quoted_currency: ~
+  submarket: random_index
```
Comment (junbon-binary) [inline share/underlyings.yml:2809]: you've created `jump_index` as submarket in `perl-Finance-Asset` but you're not using it?

File: share/underlyings.yml (hunk 1 above)
Comment (yngshan-binary) [reply share/underlyings.yml:2800]: I think you can put a comment that 10 is referring to the volatility. So that reader not confuse with jump frequency

Comment (arash-binary) [reply share/underlyings.yml]: Renamed to "Volatility 10 Jump diffusion"

File: share/underlyings.yml (hunk 2 above)
Comment (arash-binary) [reply share/underlyings.yml]: You're right. Fixed.

Comment (arash-binary) [reply share/underlyings.yml]: Although, it's actually kinda counter-intuitive: `perl-Finance-Underlying` is our public repo and Asset is private; So it does not make much sense that in Underlying we depend on something in Asset

**Review by junbon-binary — APPROVED** [review]

**Pull request merged**